go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -512,8 +535,8 @@ func (tsv *TabletServer) fullStart() (err error) {
 	}
 	c.Close()
 
-	if err := tsv.se.Open(); err != nil {
-		log.Errorf("Could not load schema, but starting the query service anyways: %v", err)
+	if err := tsv.sh.Open(); err != nil {
```

Comment:
I'm having second thoughts about this approach. Maybe we should have encapsulated sh inside se. But let's leave this as is for now. We can revisit this next time we get into this code.

For now, can you add a comment here saying that `sh` opens and closes `se`?